ONSdigital/eq-survey-runner
# EAR 218 - Cookie consent banner

Added cookie banner
To Test
Run Yarn compile to build static content.
Spin up using docker-compose up

Test out setting and unsetting the cookie preferences and check if GTM analytics code is removed in source.
